api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/orchestrator/UpdateStudentDownstreamOrchestrator.java

```diff
@@ -58,22 +69,110 @@ public void updateStudent(final Event event, final SdcSagaEntity saga, final Upd
         this.getSagaService().updateAttachedSagaWithEvents(saga, eventStates);
 
         final Student studentDataFromEventResponse = this.restUtils.getStudentByPEN(UUID.randomUUID(), updateStudentSagaData.getAssignedPEN());
-        studentDataFromEventResponse.setUpdateUser(ApplicationProperties.STUDENT_DATA_COLLECTION_API);
-        studentDataFromEventResponse.setMincode(updateStudentSagaData.getMincode());
-        studentDataFromEventResponse.setLocalID(updateStudentSagaData.getLocalID());
-        studentDataFromEventResponse.setGradeCode(updateStudentSagaData.getGradeCode());
-        updateGradeYear(studentDataFromEventResponse, updateStudentSagaData);
-        updateUsualNameFields(studentDataFromEventResponse, updateStudentSagaData);
-        studentDataFromEventResponse.setPostalCode(updateStudentSagaData.getPostalCode());
-        studentDataFromEventResponse.setHistoryActivityCode("REQMATCH");
-
-        final Event nextEvent = Event.builder().sagaId(saga.getSagaId())
-                .eventType(UPDATE_STUDENT)
-                .replyTo(this.getTopicToSubscribe())
-                .eventPayload(JsonUtil.getJsonStringFromObject(studentDataFromEventResponse))
-                .build();
-        this.postMessageToTopic(STUDENT_API_TOPIC.toString(), nextEvent);
-        log.debug("message sent to STUDENT_API_TOPIC for UPDATE_STUDENT Event.");
+        final List<SdcSchoolCollectionStudentEntity> otherStudentsWithSameAssignedID = sdcSchoolCollectionStudentRepository.findAllDuplicateStudentsByCollectionID(updateStudentSagaData.getCollectionID(), Collections.singletonList(updateStudentSagaData.getAssignedStudentID()));
+
+        if (updateStudentSagaData.getCollectionTypeCode().equalsIgnoreCase(CollectionTypeCodes.JULY.getTypeCode()) ||
```

Comment:
I think there's a logic issue here; are we not ignoring all students in the July collection?

Reply:
Good question - I was following what was said in the ticket (https://eccbc.atlassian.net/jira/software/c/projects/EDX/boards/17?selectedIssue=EDX-2964) - "In July – do not change the school of record.". But, in hindsight, my application of this doesn't make any sense. Are we not supposed to do any updates to the student api after the july collection?